app/js/lib/foxbox/foxbox.js

```diff
@@ -368,7 +368,15 @@ export default class Foxbox extends Service {
   getServices() {
     return this[p.db].getServices()
       .then(services => {
-        return services.map(service => service.data);
+        return services.map(service => {
+          // Some service don't have name, but can have product_name instead,
+          // eventually we should get rid of this line and use unified service
+          // label field provided by taxonomy.
+          service.data.properties.name = service.data.properties.name ||
```

Comment:
There is a high chance (at the moment I think it's 100% chance) that OpenZWave devices won't have name, but there is a much higher chance that they we'll have product name. It's harder to change at the server side before demo so we agreed with @julienw to fix this on the client side for now.